src/solvers/refinement/string_constraint_generator_comparison.cpp

```diff
@@ -130,14 +133,15 @@ exprt string_constraint_generatort::add_axioms_for_equals_ignore_case(
   implies_exprt a1(eq, s1.axiom_for_has_same_length_as(s2));
   axioms.push_back(a1);
 
-  symbol_exprt qvar=fresh_univ_index("QA_equal_ignore_case");
+  symbol_exprt qvar=fresh_univ_index("QA_equal_ignore_case", index_type);
   exprt constr2=character_equals_ignore_case(
     s1[qvar], s2[qvar], char_a, char_A, char_Z);
   string_constraintt a2(qvar, s1.length(), eq, constr2);
   axioms.push_back(a2);
 
-  symbol_exprt witness=fresh_exist_index("witness_unequal_ignore_case");
-  exprt zero=from_integer(0, get_index_type());
+  // TODO: we should have to give the type of witness
```

Comment:
sorry this shouldn't be there, the call the index_type has been added but the TODO wasn't removed